docs/resources/network.md

```diff
@@ -37,31 +37,34 @@ resource "upcloud_router" "example_router" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Required Attributes
 
-- `ip_network` (Block List, Min: 1, Max: 1) A list of IP subnets within the network (see [below for nested schema](#nestedblock--ip_network))
-- `name` (String) A valid name for the network
+- `name` (String) Name of the network.
 - `zone` (String) The zone the network is in, e.g. `de-fra1`. You can list available zones with `upctl zone list`.
 
-### Optional
+### Optional Attributes
 
-- `router` (String) The UUID of a router
+- `router` (String) UUID of a router to attach to this network.
+
+### Blocks
+
+- `ip_network` (Block List) A list of IP subnets within the network (see [below for nested schema](#nestedblock--ip_network))
```

Comment:
Is there any way to mention the minimum (and maximum) amount here? Having the "min 1" would implicate it is a required block.

Reply:
Would `IP subnet within the network. Network must have exactly one IP subnet.` be a good description?

Reply:
That sounds good.